src/pages/layer-2.js

```diff
@@ -0,0 +1,1097 @@
+  display: flex;
+  justify-content: center;
+  align-content: center;
+  margin-bottom: 4rem;
+  @media (max-width: ${({ theme }) => theme.breakpoints.m}) {
+    flex-direction: column;
+  }
+`
+
+const StatBox = styled.div`
+  display: flex;
+  flex-direction: column;
+  flex: 33%;
+  padding: 0 20px;
+  text-align: center;
+  align-content: center;
+  justify-content: center;
+  @media (max-width: ${({ theme }) => theme.breakpoints.m}) {
+    flex: 100%;
+  }
+`
+
+const StatPrimary = styled.p`
+  font-weight: bold;
+  font-size: ${(props) => props.theme.fontSizes.xl};
+  color: ${(props) => props.theme.colors.primary};
+  font-family: monospace;
+`
+
+const StatDescription = styled.p`
+  opacity: 0.8;
+  margin: 0;
+`
+
+const StatDivider = styled.div`
+  border-left: 1px solid ${({ theme }) => theme.colors.homeDivider};
+  max-height: 100px;
+  @media (max-width: ${({ theme }) => theme.breakpoints.m}) {
+    border-left: none;
+    border-bottom: 1px solid ${({ theme }) => theme.colors.homeDivider};
+    width: 100%;
+    height: 0%;
+    margin: 2rem 0;
+  }
+`
+
+const Layer2Page = ({ data }) => {
+  const intl = useIntl()
+
+  const [tvl, setTVL] = useState("-")
+  const [percentChangeL2, setL2PercentChange] = useState("-")
+  const [averageFee, setAverageFee] = useState("-")
+
+  useEffect(() => {
+    const fetchL2Beat = async () => {
+      try {
+        const l2BeatData = await getData("https://l2beat.com/api/tvl.json")
+        // formatted TVL from L2beat API formatted
+        setTVL(
+          new Intl.NumberFormat(intl.locale, {
+            style: "currency",
+            currency: "USD",
+            notation: "compact",
+            minimumSignificantDigits: 2,
+            maximumSignificantDigits: 3,
+          }).format(l2BeatData.data[l2BeatData.data.length - 1][1])
+        )
+        // Calculate percent change ((new value - old value) / old value) *100)
+        setL2PercentChange(
+          (
+            ((l2BeatData.data[l2BeatData.data.length - 1][1] -
+              l2BeatData.data[l2BeatData.data.length - 31][1]) /
+              l2BeatData.data[l2BeatData.data.length - 31][1]) *
+            100
+          ).toFixed(2)
+        )
+      } catch (error) {
+        console.error(error)
+      }
+    }
+
+    fetchL2Beat()
+
+    const fetchCryptoStats = async () => {
+      try {
+        // Average eth transfer fee from L2's supported by cryptostats API
+        const feeData = await getData(
+          "https://api.cryptostats.community/api/v1/l2-fees/feeTransferEth?metadata=false"
+        )
+        const feeAverage =
+          feeData.data.reduce(
+            (acc, curr) => (acc += curr.results.feeTransferEth),
+            0
+          ) / feeData.data.length
+        setAverageFee(feeAverage.toFixed(2))
+      } catch (error) {
+        console.error(error)
+      }
+    }
+    fetchCryptoStats()
+  }, [])
+
+  const heroContent = {
+    title: "Layer 2",
+    header: "Layer 2",
```

Comment:
I suspect we could add a more useful title/header combo here (currently both `Layer 2` - a bit repetitive), e.g. like on Stablecoins page:

![Image 2022-03-30 at 3 39 02 PM](https://user-images.githubusercontent.com/8097623/160942611-d0acafe4-3755-45ed-bd54-3ca82348563f.jpg)

Any thoughts @minimalsm @konopkja?

Reply:
Some ideas @samajammin @corwintines @konoleoda. 

I quite like the last one (_"Ethereum for everyone"_) because it's pithy, a little abstract, and whimsical but at a very high level describes the goal of Layer 2.

<img width="1255" alt="Screenshot 2022-03-31 at 10 49 55" src="https://user-images.githubusercontent.com/62268199/161028727-fb742967-aa24-439b-95df-28e1bd14ceb9.png">
<img width="1439" alt="Screenshot 2022-03-31 at 10 50 10" src="https://user-images.githubusercontent.com/62268199/161028746-44b63a08-b81c-4b7e-9052-c1b2cca68833.png">
<img width="1385" alt="Screenshot 2022-03-31 at 10 58 42" src="https://user-images.githubusercontent.com/62268199/161029688-529e34af-838e-4ad6-8b0b-2f1fe6aa60bc.png">
<img width="1306" alt="Screenshot 2022-03-31 at 10 51 33" src="https://user-images.githubusercontent.com/62268199/161028752-e34ab68f-ad9c-4aee-92c9-9cc4e9e83f50.png">
<img width="1402" alt="Screenshot 2022-03-31 at 10 58 01" src="https://user-images.githubusercontent.com/62268199/161029665-321f7677-3b38-40c1-b248-031b3eeb996d.png">

<img width="1446" alt="Screenshot 2022-03-31 at 10 58 56" src="https://user-images.githubusercontent.com/62268199/161029691-2fc9849b-5a02-46e9-b2f3-20249c465ea9.png">

<img width="1368" alt="Screenshot 2022-03-31 at 10 53 17" src="https://user-images.githubusercontent.com/62268199/161028758-f9a07359-3b45-47b3-bda6-8e87be49ae2d.png">
<img width="1419" alt="Screenshot 2022-03-31 at 10 53 40" src="https://user-images.githubusercontent.com/62268199/161028762-52ac1d42-7046-4682-9cdd-9a288981c797.png">